app/services/container_dashboard_service.rb

```diff
@@ -163,7 +165,7 @@ def build_provider_status(provider_type)
       :count      => 0,
       :typeName   => _(provider_type.to_s),
       :iconImage  => provider_fileicon ? ActionController::Base.helpers.image_path(provider_fileicon) : nil,
-      :statusIcon => provider_status_icon
+      :statusIcon => provider_status_icon || nil
```

Comment:
Is this `|| nil` really necessary?

Reply:
@skateman removed ` || nil ` it seems to work well without that as well. I needed to set that before your recent change.